source/ed/connectors/osm_tags_reader.cpp

```diff
@@ -205,6 +206,17 @@ std::bitset<8> parse_way_tags(const std::map<std::string, std::string>& tags) {
                 foot = foot_allowed;
             }
         }
+
+        // We do not want to autocomplete on public transport objects
+        if (key == "public_transport") {
```

Comment:
What about excluding all object with a key="public_transport". None of the values seem to be interesting to us - https://wiki.openstreetmap.org/wiki/Key:public_transport